components/restapi/core/accounts.go

```diff
@@ -19,17 +19,17 @@ import (
 func congestionForAccountID(c echo.Context) (*apimodels.CongestionResponse, error) {
 	accountID, err := httpserver.ParseAccountIDParam(c, restapipkg.ParameterAccountID)
 	if err != nil {
-		return nil, err
+		return nil, ierrors.Wrapf(echo.ErrBadRequest, "failed to parse account ID %s: %s", c.Param(restapipkg.ParameterAccountID), err)
```

Comment:
no need to wrap errors that come from the `httpserver` package since those are already properly wrapped with underlying http errors.
These changes can be reverted in the whole PR.